applications/StructuralMechanicsApplication/CMakeLists.txt

```diff
@@ -196,4 +196,6 @@ install(TARGETS KratosStructuralMechanicsCore DESTINATION libs )
 install(TARGETS KratosStructuralMechanicsApplication DESTINATION libs )
 
 # Add to the KratosMultiphisics Python module
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/StructuralMechanicsApplication.py" DESTINATION KratosMultiphysics )
+install(DIRECTORY DESTINATION "KratosMultiphysics/StructuralMechanicsApplication")
+
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/StructuralMechanicsApplication.py" DESTINATION "KratosMultiphysics/StructuralMechanicsApplication" RENAME "__init__.py")
```

Comment:
same as above, I am not sure if this works in Win (using `/` for separating the path)
=> if not I think it can be easily fixed